### PR TITLE
Issue #33:  Change yaml linting line length to 120 

### PR DESCRIPTION
**Pull request by SonOfLope**
- Adding config file for yamllint with rule for line length set to 120
- Fix all yaml violations

### CODE REVIEW

**Review by ThomasCardin — APPROVED** [review]
LGTM!

File: kubernetes/apps/nachet/nachet-backend-deployment.yml [hunk 1]
```diff
@@ -38,21 +38,21 @@ spec:
...
-            port: 8080
-          initialDelaySeconds: 60
-          periodSeconds: 10
+        - name: nachet-backend
+          image: ghcr.io/ai-cfia/nachet-backend:48
+          imagePullPolicy: Always
+          command: ["/bin/sh", "-c"]
+          args:
+            - >
+              cp /vault/secrets/.env . && hypercorn -b :8080 app:app
```
Comment (rngadam) [inline kubernetes/apps/nachet/nachet-backend-deployment.yml:47]: Can you clarify why the environment variables cannot be passed to the container and why we have to break the container abstraction (directly configuring the Asynchronous Server Gateway Interface webserver to setup the environment? I've asked why this was necessary previously but haven't heard a good explanation to why this is preferable.

**Review by rngadam — CHANGES_REQUESTED** [review]

**Review by rngadam — CHANGES_REQUESTED** [review]
There seems to be quite a bit more changes than just changing the line length to 120.

**Comment by SonOfLope** [comment]
> There seems to be quite a bit more changes than just changing the line length to 120.

We are resolving all yaml violations that we didnt resolved until we changed the linee length ruling

File: kubernetes/apps/nachet/nachet-backend-deployment.yml (hunk 1 above)
Comment (SonOfLope) [reply kubernetes/apps/nachet/nachet-backend-deployment.yml:47]: This PR is only solving yaml linting violatiions and applying the new line length ruling to 120. Can we discuss this in an issue ? From what I understood we want to address this issue [here](https://github.com/ai-cfia/howard/issues/27)

**Review by rngadam — APPROVED** [review]

**Pull request merged**